[제정욱] Mission 6: 로그인 JWT 발급

jeajeongwook opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
(여기에 수정 사항에 대한 간략한 한줄 요약/제목 작성해 주세요.)
- 로그인시 JWT 발급되게 코드 변경
		 
## 수정 사항들 자세한 내용
(여기에 수정 사항에 대한 자세한 내용을 작성해 주세요.)
- 암호화된 비밀번호 끌어와서 로그인 성공 하는점 확인
- 로그인 성공시 http 200 코드와 함께 유저의 db내의 id가 담긴 jwt token을 반환
- 
## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)
- 이메일 validation을 함수화할 수는 없나요? (예시) 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
조건을 만족한다면 각 빈칸에 x 표시 해주세요
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

Commit:
Apply JWT

ezer-1 commented on students/jeongwook/users/views.py:
```diff
@@ -42,11 +43,13 @@ def post(self, request):
-            
-            return JsonResponse({"message":"SUCCESS"}, status=200)
+            if not bcrypt.checkpw(password.encode('UTF-8'), user.password.encode('UTF-8')):
+                return JsonResponse({"massage": "INVALID_USER"}, status=401)
```
오타 확인부탁드립니다.

jiggyjiggy replied:
ㅋㅋㅋ 마사지 시원하겠네요 

정욱님의 의도치 않은 오타 덕분에 빵터졌습니다 
기운 찬 하루 될 것 같습니다
2팀 모두 화이팅!

jeajeongwook replied:
ㅋㅋㅋ 영어 너무 약캐요 ㅠㅠ

jiggyjiggy commented:
LGTM~~! 
깔끔하게 잘 작성하신 것 같습니다!

Commit:
5 / 5000발음듣기복사하기필기인식기번역하기자동완성자동완성
영어열기/닫기 아이콘
Typing correctio5 / 5000발음듣기복사하기필기인식기번역하기자동완성자동완성
영어열기/닫기 아이콘
Typing correction
git commit -m

Commit:
Typing correction

BDjaekwanee commented:
LGTM~~!!!!!! 문제 없어 보입니다!
